src/components/DonateButton.jsx

```diff
@@ -263,12 +263,12 @@ class DonateButton extends Component {
                   onChange={(name, value) => this.setState({ mewAmount: value })}
                   placeholder="10"
                   validations={{
-                    lessThan: wallet.getBalance() - 0.5,
-                    greaterThan: 0.00000000009,
+                    lessThan: wallet.getBalance() + (gas * 1000000000),
```

Comment:
okay, a couple of things.

1. do you mind re-naming `gas` to `gasPrice`. I know you didn't name gas, but this would make everything more clear.
2. `1000000000` wasn't clear to me what it was. But I think you are converting the gasprice from `gwei` to `wei` which is correct. To make this more clear can you either use use a variable like `gwei = 1000000000` or preferably use `Web3.utils` `utils.toWei(gas, 'gwei')`.
3. we also need to take into account the amount of gas used by the tx. In `componentDidMount` we call `estimateGas`. We can use that value ( with a default of 600k incase the estimate gas fails) and multiply it by the gasPrice in wei. 600k is the default b/c w/ our current contracts, ~496k is the most gas used. This gives us a bit of wiggle room.
4. I think the gas needs to be subtracted from the walletBalance, not added.
5. `wallet.getBalance()` returns eth, but you are adding wei, we need to convert to the same units for any adding/subtracting, then to eth for the validator.